syntax = "proto2";
package spider;

message consumer_register_resp {
	required int32 id = 1;
	required int32 res = 2;
}
